### ISSUE TITLE
We have a new property P5324  

### ISSUE THREAD

**Issue by salgo60** [maxlath/hub#9]
Looks like it doesnt work yet 
https://tools.wmflabs.org/hub/Q10728061?property=P5324 ==> _"unknown property id"_

Is there anything we can do to let the property P5324 in  ;-)

Regards
Magnus Sälgö
Stockholm, Sweden
Wikidata [salgo60](https://www.wikidata.org/wiki/User:Salgo60)

**Comment by maxlath** [comment]
that should already be [done automatically every 24h](https://github.com/maxlath/hub/blob/master/lib/refresh_properties_once_a_day.js), but that weirdly fails with some somewhat cryptic error:
```
(node:432) fs: re-evaluating native module sources is not supported. If you are using the graceful-fs module, please update it to a more recent version.
```
Restarting the server manually does solve the issue though... until new properties are created

**Comment by salgo60** [comment]
Thanks it works **BUT** ;-) I changed the formatter URL string on [https://www.wikidata.org/w/index.php?diff=698005311&oldid=697278382&title=Property%3AP778&type=revision P778] I guess we need a new restart ?!?!?!

Its really cool functionality

- If a site have the **Qnumber** for a Swedish region then by using the **hub** we can 
-  [Jump to the Swedish National Archives](https://sok.riksarkivet.se/?postid=ArkisRef%20SE/HLA/1010201) using P5324 and see online all churchbooks for the last 400 years
-  Jump to the National Statistics bureau of Sweden and see [a Map of how its borders are today](http://geodata.scb.se/reginawebmap/main/webapp/?typ=forsamling&a=0000&f=018013)  using P778

**Comment by maxlath** [comment]
restarted!

**Comment by BraveheartWikimedia** [comment]
Hi @maxlath, is there now a scheduled restart of the hub to deal with this issue? Or does this still require a manual restart?

I've been trying to figure out why the South Tyrolean Monument ID P8543 doesn't work with the hub, but the previous comments here explain it. Example link: https://hub.toolforge.org/P8543:14344?site=wikidata

**Comment by maxlath** [comment]
It was still requiring a manual restart, which I just did, but also I hope the new version to now really update daily, following commit c985858 . Let see how it goes in the coming days

**Comment by mormegil-cz** [comment]
Apparently, this still does not work? There is [P11031](https://www.wikidata.org/wiki/Property:P11031) (created on 2022-09-02), which has been correctly added (automatically, I guess) to `properties.js`, but [Hub still returns](https://hub.toolforge.org/P11031:1) “unknown property id” for it.

**Comment by maxlath** [comment]
@mormegil-cz thanks for reporting! There was again an issue with the daily refresh, which should have been fixed by c0740d9. Let me know if it's failing again.
